lib/config-util.js

```diff
@@ -18,18 +18,35 @@ function guessCWD (cwd) {
   return cwd
 }
 
+function moduleExists (name) {
+  try {
+    return typeof require.resolve(name) !== 'undefined'
+  } catch (e) { return false }
+}
+
 Config.loadConfig = function (argv, cwd) {
   const rcOptions = [
     argv.nycrcPath || '.nycrc',
     '.nycrc.json',
+    '.nycrc.yml',
+    '.nycrc.yaml',
     'nyc.config.js'
   ]
   const rcPath = findUp.sync(rcOptions, { cwd: guessCWD(cwd) })
   let config = {}
 
   if (rcPath) {
-    if (rcPath.toLowerCase().endsWith('.js')) {
+    const rcPathLower = rcPath.toLowerCase()
+    if (rcPathLower.endsWith('.js')) {
       config = require(rcPath)
+    } else if (/.+\.y(a?)ml$/.test(rcPathLower)) {
+      if (moduleExists('js-yaml')) {
```

Comment:
It might be worth check for `yaml` as well?  It's pretty nice.  The only difference is that you'd use `require('yaml').parse()` instead of `require('js-yaml').load()`.

Reply:
cc @bcoe